### PR TITLE
[Features Redesign] Update tab styling, layout

### PR DESCRIPTION
**Pull request by Harshita-Kanal**
- [x] Remove old comparison chart
- [x] Style to buttons to other tools
Features-tab

Signed-off-by: Harshita-Kanal <harshita.kgv@gmail.com>

### CODE REVIEW

File: content/features.md [hunk 1]
```diff
@@ -49,10 +49,6 @@ Leverage production-ready buildpacks maintained by the community.
 
 </div>
 
-## Comparison
-
-{{< comparison-table >}}
```
Comment (jromero) [inline content/features.md:54]: Can we remove the file associated with this as part of this PR as well?

File: themes/buildpacks/layouts/shortcodes/in-depth-comparison-table.html [hunk 2]
```diff
@@ -453,6 +457,44 @@
...
+          <td class="border-left border-right">
+            <div style="overflow-x: auto; text-align: left">
+              <p class="text-pink font-weight-bold">Governance</p>
+              CNCF
+            </div>
+          </td>
+          <td class="comp-dockerfiles">
+            <div style="text-align: left">
+              <p></p>
+              <br />
```
Comment (jromero) [inline themes/buildpacks/layouts/shortcodes/in-depth-comparison-table.html:471]: Why do we need this empty paragraph (`<p>`) and  break (`<br/>`)?

Comment (Harshita-Kanal) [reply themes/buildpacks/layouts/shortcodes/in-depth-comparison-table.html:471]: That was for the layout, to align the start of text with the left column, i'll make this cleaner with CSS :)

**Comment by Harshita-Kanal** [comment]
Made the changes :)

**Review by jromero — APPROVED** [review]

**Pull request merged**